PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.java

```diff
@@ -56,10 +56,19 @@ void sendRequest(final Context context, final PayPalRequest payPalRequest,
                         ? SETUP_BILLING_AGREEMENT_ENDPOINT : CREATE_SINGLE_PAYMENT_ENDPOINT;
                 String url = String.format("/v1/%s", endpoint);
 
-                String requestBody =
-                        payPalRequest.createRequestBody(configuration,
-                                braintreeClient.getAuthorization(),
-                                successUrl, cancelUrl);
+                String universalLink = null;
```

Comment:
I'm not sure this PR is necessary - we already include the app link return URL in lines 39/40. App links are the only available integration pattern in v5.